README.md

```diff
@@ -47,15 +47,16 @@ apt-get install xrdp
 
 ### RedHat / CentOS / Fedora
 ```bash
-yum -y install xrdp
-dnf -y install xrdp
-
-After installing Xrdp in Centos, RHEL or Fedora  you need to adjust the firewall 
+dnf install xrdp
+```
 
-# firewall-cmd --permanent --zone=public --add-port=3389/tcp
-# firewall-cmd --reload
+Older distribution should use `yum` instead of `dnf`. To allow outside
```

Comment:
I don't think the words "Older distribution" is not always correct.   `dnf` replaced `yum` since Fedora 22.  RHEL 7.2 is released later than Fedora 22.   It's a trivial issue though.